products/sle/main.pm

```diff
@@ -497,11 +501,13 @@ sub load_inst_tests() {
             loadtest "installation/disk_space_fill";
         }
     }
-    if (get_var('SCC_REGISTER', '') eq 'installation') {
-        loadtest "installation/scc_registration";
-    }
-    else {
-        loadtest "installation/skip_registration";
+    if (!check_var('VERSION', '15')) { # No registration in SLE 15 atm - rbrown 04/07/17
+        if (get_var('SCC_REGISTER', '') eq 'installation') {
```

Comment:
it's a good question, but I was preserving the existing behaviour - didn't want to risk changing anything for SLE 12 SP3 and earlier, just wanted to eliminate the loading of the modules for 15